forwards_paper.tex

```diff
@@ -903,59 +903,80 @@ \subsubsection*{Estimates of run-time complexity}
 
 
 %%%%%%
-\subsection*{Overview of the API}
+\subsection*{Implementation overview}
 
-These tools are implemented through the \msprime{} Python API,
-which provides a powerful platform for working with tree topology and mutation data.
-The new portions of \msprime{} that we cover here
-are dedicated to tree sequence input and output using simple tables of data,
-so we refer to this as the `Tables API'.
-The four key tables: nodes, edges, sites and mutations, are described above.
+The facilities for working with succinct tree sequences are implemented as part
+of the \msprime\ Python API, which provides a powerful platform for processing
+tree topology and mutation data. The new portions of \msprime{} that we discuss
```

Comment:
when and where do you use `\msprime\` vs `\msprime{}`? need to catch up on .tex tricks

Reply:
It doesn't matter --- I just use the trailing backslash out of habit.